osu.Game/Screens/OnlinePlay/Playlists/PlaylistsRoomSettingsOverlay.cs

```diff
@@ -326,18 +351,9 @@ private void populateDurations(ValueChangedEvent<APIUser> user)
                     TimeSpan.FromDays(3),
                     TimeSpan.FromDays(7),
                     TimeSpan.FromDays(14),
+                    TimeSpan.FromDays(days_in_month),
+                    TimeSpan.FromDays(days_in_month * 3),
```

Comment:
I'd hope these had some hinting in the dropdown itself, whether that be a `*` next to it or ` (supporter)` or something else.

Reply:
Other supporter features never hint until it's clicked (i.e. beatmap listing filters and country/friend/mod leaderboards) so those need to be reconsidered too.